schemas/com.snowplowanalytics.snowplow/media_player_event_play/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event sent when the player changes state to playing from previously being paused.",
```

Comment:
Do we fire this implicitly when playback starts for the first time?

Reply:
We don't distinguish between starting play for the first time or resuming playing after being paused. Do you suggest differentiating the two cases or just improving the description?